src/core_ct/core.py

```diff
@@ -282,3 +282,39 @@ def chunk(self, x1=0, y1=0, z1=0, x2=None, y2=None, z2=None) -> Core:
 
         new_core = Core(self.pixel_array[x1:x2, y1:y2, z1:z2], self.pixel_dimensions)
         return new_core
+
+    def join(self, core: Core, axis: int = 0) -> Core:
+        """
+        Join a core to the current core on specified axis.
+
+        Arguments:
+        ---------
+            core: the `Core` object to join with the current core
+            axis: the axis to join the cores on
+
+        Returns:
+        -------
+            New core object made up of the two joined arrays
+
+        Raises:
+        ------
+            Exception if axis is a value other than 0, 1, or 2
```

Comment:
Potentially change these from `Exception` to `ValueError` to be more specific.

Reply:
Updated all the `Exeception`s to be `ValueError`s